Dockerfile-infra

```diff
@@ -4,17 +4,19 @@ FROM ruby:${ruby_version}-alpine${alpine_version}
 
 ARG covalence_version
 ARG dumbinit_version
-ARG gemfury_token
 ARG gosu_version
+ARG sops_version
 
 ENV COVALENCE_VERSION $covalence_version
 ENV DUMBINIT_VERSION $dumbinit_version
 ENV GOSU_VERSION $gosu_version
 ENV GOSU_KEY B42F6819007F00F88E364FD4036A9C25BF357DD4
+ENV SOPS_VERSION $sops_version
 
 RUN set -ex; \
   \
   fetchDeps=' \
+    ansible \
```

Comment:
was going to pull it in with pip but it didn't pull in the required dependencies automatically, i figured this should be easy and sufficient.